### DISCUSSION TITLE
Difference between Load Power, Inverter Power, Battery Power

### DISCUSSION THREAD

**Discussion by LionBS76** [vladyspavlov/esphome-must-inverter#7] [General]
Hello!

Have Must PV18-3224 inverter & ESPHome (esp01)  with this project yaml config & Home Assistant.

When analizing data from some sensors in Home Assistant I see difference between:
- Load Power & Inverter Power & Battery Power

What exactly this sensors mean ?
As I can assume: 
Load Power + Inverter block consumption = Inverter Power
Inverter Power + Some other block consumption = Battery Power

From history graph difference between them abou 48-50 W.

Any idea ?

![image](https://github.com/user-attachments/assets/e006c859-b120-43d7-b0e4-bf932f33e991)

**Comment by vladyspavlov** [comment]
It's not documented, but as i know:
Battery power - load from or to the battery (when grid or solar is not available it's discharge power and charger power).
Inverter power - load power + self-consumption
Load power - clear power of your load

These sensors are not very accurate at all.
